opts/file.go

```diff
@@ -58,8 +61,12 @@ func parseKeyValueFile(filename string, emptyFn func(string) (string, bool)) ([]
 			}
 
 			if len(data) > 1 {
-				// pass the value through, no trimming
-				lines = append(lines, fmt.Sprintf("%s=%s", variable, data[1]))
+				// pass the value through, trimming leading and trailing quotes
+				value := data[1]
+				value = strings.TrimLeft(value, quotes)
+				value = strings.TrimRight(value, quotes)
```

Comment:
We should probably make sure that we only trim the "outer" quotes, and only if "left" and "right" have the same quote; I was writing along some ideas for that, and push it as a PR against your branch https://github.com/photra/cli/pull/2

Overall, we should look if there's potential side-effects / changes in behavior because of this though 😅 - perhaps there's other expectations (such as, when using single quotes, don't expand other env-vars inside the value).

I know the original design for the env-file was explicitly to only split on `=`, and nothing else.

Reply:
Given that `parseKeyValueFile()` is also used in other parts of the code, we may need to check if we want the behavior to change for those areas as well

Reply:
That's a good point! I checked `parseKeyValueFile()` and after evaluating `readKVStrings()` and `ParseEnvFile()`, syntactically there shouldn't be side effects given their signatures.

Additionally, I read our [documentation](https://docs.docker.com/engine/reference/commandline/run/#set-environment-variables--e---env---env-file) on `--env-file`, and it looks like we do not perform any operations on the variable or its value. Where we do this is within the [Dockerfile](https://docs.docker.com/engine/reference/builder/#run) when using shell form: 

`"When using the exec form and executing a shell directly, as in the case for the shell form, it is the shell that is doing the environment variable expansion, not docker."`

I think we should be OK on `env-file` behavior, but we should look closer for `Dockerfile` image building. I'll update here if I find anything. Let me know if you found something!